pkg/model/planner.go

```diff
@@ -50,6 +50,40 @@ func (r *Run) Job() *Job {
 	return r.Workflow.GetJob(r.JobID)
 }
 
+// Helper function for FixIfstatement
+func FixIfStatement1(val string, lines [][][]byte, l int) (string, error) {
+	if val != "" {
+		line := lines[l-1][0]
+		outcome := regexp.MustCompile(`\s+if:\s+".*".*`).FindSubmatch(line)
```

Comment:
is:
```
if:"foo"
```

not valid?

Reply:
While I did my bugtesting, I both checked the documentation and tried each if statement combination that I could think of. The if:"foo" was in the incorrect format according to github action when i tested it. Which is why I demand the \s+ before the "foo"